myhpi/templates/base.html

```diff
@@ -76,13 +81,20 @@
                 <div id="nav-level-2" class="nav-level"></div>
 
                 <div id="nav-level-right"
-                    class="nav-level d-flex flex-column flex-grow-1 align-items-stretch align-items-xl-end">
+                    class="nav-level d-flex flex-column flex-grow-2 align-items-stretch align-items-xl-end">
```

Comment:
Why is this needed?

Reply:
This might be a relict of changes no longer needed. Will be changed in the next commit.